projects/ui/src/lib/components/po-tag/po-tag-base.component.ts

```diff
@@ -106,7 +169,7 @@ export class PoTagBaseComponent {
   @Output('p-click') click?: EventEmitter<any> = new EventEmitter<PoTagItem>();
 
   get iconFromType() {
-    switch (this.type) {
+    switch (this._type) {
```

Comment:
Sugestão:

Utilizar this.type, pois pode ser que em algum momento tenha um tratamento no get do type e o mesmo não sera a 'reconhecido'.